omnigibson/scene_graphs/graph_builder.py

```diff
@@ -0,0 +1,272 @@
+import itertools
+import os
+
+import networkx as nx
+import numpy as np
+from PIL import Image
+from matplotlib import pyplot as plt
+
+from omnigibson import object_states
+from omnigibson.sensors import VisionSensor
+from omnigibson.object_states.factory import get_state_name
+from omnigibson.object_states.object_state_base import AbsoluteObjectState, BooleanStateMixin, RelativeObjectState
+from omnigibson.utils import transform_utils as T
+
+DRAW_EVERY = 1
+
+
+def get_robot(scene):
+    assert len(scene.robots) == 1, "Exactly one robot should be available."
+    return scene.robots[0]
+
+
+def get_robot_bbox(robot):
+    # The robot doesn't have a nicely annotated bounding box so we just return AABB for now.
+    return T.pose2mat((robot.aabb_center, [0, 0, 0, 1])), robot.aabb_extent
+
+
+def get_robot_to_world_transform(robot):
+    # TODO: Maybe do this in eye frame
+    robot_to_world = robot.get_position_orientation()
+
+    # Get rid of any rotation outside xy plane
+    robot_to_world = T.pose2mat((robot_to_world[0], T.z_rotation_from_quat(robot_to_world[1])))
+
+    return robot_to_world
+
+
+def get_unary_states(obj, only_true=False):
+    states = {}
+    for state_type, state_inst in obj.states.items():
+        if not issubclass(state_type, BooleanStateMixin) or not issubclass(state_type, AbsoluteObjectState):
```

Comment:
can we improve the semantics here for interpretability? e.g.: `get_boolean_unary_states`